make-indexes.sh

```diff
@@ -215,6 +201,16 @@ fi
 
 # These excluded repos don't have branch and releases indexes
 if [[ ! "$AMWA_ID" == "SPECS" && ! "$AMWA_ID" == "NMOS" && ! "$AMWA_ID" == "BCP-002" && ! "$AMWA_ID" == "BCP-003" ]]; then
+    echo Adding releases index...
+    INDEX_RELEASES="releases/index.md"
+    echo "## Published Releases" > "$INDEX_RELEASES"
+    echo -e "\n##  Published Releases" >> "$INDEX"
+    for release in $(ls -r1 releases/); do
```

Comment:
To avoid the  `SC2045: Iterating over ls output is fragile. Use globs.` lint error

```suggestion
    for release in $(cd releases && echo -- * | awk '{ for (i=NF; i>=1; i--) print $i}'); do
```

Reply:
The `ls` command is easier to read and `-r1` can only be intended for machine parsing, but the cd/echo-glob/awk-loop combo is at least still an intelligible one-liner. I'm OK with either.